pkg/auth/auth.go

```diff
@@ -64,9 +67,13 @@ func getUsername(usernameClaim UsernameClaimType, claims *Claims) string {
 	return claims.Subject
 }
 
-// AuthorizationEnabled returns true if the storage system has authorization
-// and authentication enabled.
-func AuthorizationEnabled(ctx context.Context) bool {
-	_, ok := NewUserInfoFromContext(ctx)
-	return ok
+// Enabled returns whether or not auth is enabled.
+func Enabled() bool {
```

Comment:
I wanted to keep `Enabled` inside of the `auth` package,  so I needed to introduce this flag for setting when auth is enabled or disabled. 

Before, we checked the SystemToken instance to if there was an issuer register. If auth was disabled, the issuer was empty. This method will not work anymore because I've created a new `systemtoken` package  that depends on `auth`. The old way of doing this would cause a circular dependency.